fhttp/httprunner.go

```diff
@@ -37,6 +37,7 @@ type HTTPRunnerResults struct {
 	periodic.RunnerResults
 	client   Fetcher
 	RetCodes map[int]int64
+	IPCount  map[string]int64
```

Comment:
👍 but should be in shared results ideally? (even if we start with just http runner all the runners involving network should do that - but we can postpone that and move it later (there isn't a NetworkRunnerResults that would extend periodic' or we can just assume most of fortio is about network and put it there)

Reply:
Added a todo for this